src/main/java/com/wcc/platform/service/CmsService.java

```diff
@@ -70,6 +72,19 @@ public CollaboratorPage getCollaborator() {
     }
   }
 
+  /**
+   * Write Pojo Member to JSON.
+   */
+  public Member createMember(Member member) {
+    try {
+        FileMemberRepository fileMemberRepo = new FileMemberRepository(objectMapper);
```

Comment:
This is the part I mentioned to you to use Spring to inject the correct bean. 

In your service, you would add the interface: 

private MemberRepository repository; 

// add in the constructor to autowire